static/app/components/dropdownMenuControl.tsx

```diff
@@ -8,9 +8,9 @@ import {Item, Section} from '@react-stately/collections';
 import {useMenuTriggerState} from '@react-stately/menu';
 import {MenuTriggerProps} from '@react-types/menu';
 
-import DropdownButton, {DropdownButtonProps} from 'sentry/components/dropdownButtonV2';
-import {MenuItemProps} from 'sentry/components/dropdownMenuItemV2';
-import Menu from 'sentry/components/dropdownMenuV2';
+import DropdownButton, {DropdownButtonProps} from 'sentry/components/dropdownButton';
+import {MenuItemProps} from 'sentry/components/dropdownMenuItem';
+import MenuList from 'sentry/components/dropdownMenuList';
```

Comment:
The component name should match the file name.

Reply:
yes it would be awesome if this was a named export :)